pkg/util/kubernetes/apiserver/util.go

```diff
@@ -61,3 +67,35 @@ func UnstructuredFromWPA(structIn *v1alpha1.WatermarkPodAutoscaler, unstructOut
 	unstructOut.SetUnstructuredContent(content)
 	return nil
 }
+
+func retryHandler(informers map[InformerName]cache.SharedInformer, name InformerName, config retryConfig) error {
```

Comment:
nit: I think it is enough if you use the cache.SharedInformer as an input and not  the whole map